src/main/java/de/tum/in/www1/artemis/domain/quiz/ShortAnswerSubmittedText.java

```diff
@@ -95,7 +95,7 @@ public void setSubmittedAnswer(ShortAnswerSubmittedAnswer shortAnswerSubmittedAn
      * @return boolean true if submittedText fits the restrictions above, false when not
      */
     public boolean isSubmittedTextCorrect(String submittedText, String solution) {
-        return FuzzySearch.ratio(submittedText.toLowerCase(), solution.toLowerCase()) > 85;
+        return FuzzySearch.ratio(submittedText.toLowerCase().trim(), solution.toLowerCase().trim()) > 85;
```

Comment:
I think we also need to trim on the client side when deciding whether the color is red or yellow

Reply:
https://github.com/ls1intum/Artemis/blob/develop/src/main/webapp/app/exercises/quiz/shared/questions/short-answer-question/short-answer-question.component.ts#L147 decides between green and yellow after it is already confirmed the answer is correct (so it doesn't decide between yellow and red there). That means the comparison at https://github.com/ls1intum/Artemis/blob/develop/src/main/webapp/app/exercises/quiz/shared/questions/short-answer-question/short-answer-question.component.ts#L161 fails (intentionally?) causing it to be yellow not green.

I can fix that check to trim as well, but then an answer with trailing spaces would appear as green too:
![grafik](https://user-images.githubusercontent.com/7704140/94168020-7da48800-fe8d-11ea-8d47-930d13887897.png)

Reply:
I think we should consider it as "completely correct" in case there are whitespaces at the beginning or end, so please add the trim() command to line 161.

In line 144 of the same file: can you please extract a variable for the result of `this.getSubmittedTextForSpot(spotTag)`

Reply:
Updated. I'll work on adding test cases now.